### DISCUSSION TITLE
how to update title in next/Head using persistent layout

### DISCUSSION THREAD

**Discussion by thomas159** [vercel/next.js#18765] [Help] [answered]
I'm using this layout pattern in nextjs as this seems to be the recommened way of avoiding the layout to re-render, however I can't see how I can update the page title in the head of the layout, if I put the <head><title>Page Name</title></head> into a component it renders it in the wrong place.

How can I update the title of the page using this pattern?

**_app.js**

import React from 'react' import App from 'next/app' import Layout from '../components/Layout'

```
const MyApp = ({ Component, pageProps }) => {
  return (
    <Layout>
      <Component {...pageProps} />
    </Layout>
  )
}
export default MyApp  
```

**Layout.js**

```
const Layout = ({ children }) => (
  <div className='layout'>
    <GlobalStyle />
    <Head>
      <link rel="preconnect" href="https://app.snipcart.com" />
      <link rel="preconnect" href="https://cdn.snipcart.com" />
      <link rel="stylesheet" href="https://cdn.snipcart.com/themes/v3.0.19/default/snipcart.css" />
    </Head>
    <Wrap>
    <Spacer />
    <Header />
    <MainWrap>
      {children}
    </MainWrap>
    <script async src="https://cdn.snipcart.com/themes/v3.0.19/default/snipcart.js" />
    <div hidden id="snipcart" data-api-key="" />
    </Wrap>
  </div>
)

export default Layout
```

**Comment by lfades** [comment, marked as answer]
You can use `next/head` in the page itself, so if the the `Layout` doesn't know the title or doesn't have access to it, use the page instead 👍
